source/ui/HUD.hx

```diff
@@ -0,0 +1,30 @@
+package ui;
+
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.text.FlxText;
+import flixel.util.FlxColor;
+import flixel.group.FlxGroup.FlxTypedGroup;
+
+using flixel.util.FlxSpriteUtil;
+
+
+
+class HUD extends FlxTypedGroup<FlxSprite> 
+{
+    var tokenCount:FlxText;
+
+    public function new()
+    {
+        super();
+        tokenCount = new FlxText(16, 2, 0, "0 / 5", 8);
```

Comment:
Not something that needs to change, but it would be pretty easy to make the total number of coins configurable here. Something little like that makes it easy to expand the game even further (although I understand why that wasn't a priority for this homework)